<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-SPJ27N6" failures="1" tests="21" name="Default test" time="557.54" errors="0" timestamp="2024-10-15T21:50:08 IST">
  <testcase classname="Test_Supervisor" name="verify_that_createSupervisor_with_invalid_email" time="52.003"/>
  <testcase classname="Test_Supervisor" name="Verify_that_the_full_name_field_does_not_accept_numeric_or_special_characters_accept_only_character" time="22.442"/>
  <testcase classname="Test_Supervisor" name="Verify_that_the_email_field_is_mandatory_and_displays_an_error_message_when_left_empty" time="22.271"/>
  <testcase classname="Test_Supervisor" name="Verify_that_the_full_name_field_is_mandatory_and_displays_an_error_message_when_left_empty" time="20.202"/>
  <testcase classname="Test_Supervisor" name="Verify_that_the_mobile_number_field_does_not_accept_alphabetic_or_special_characters" time="22.281"/>
  <testcase classname="Test_Supervisor" name="Verify_that_the_mobile_number_field_accepts_different_formats_based_on_the_selected_country_code" time="24.751"/>
  <testcase classname="Test_Supervisor" name="Verify_that_the_user_can_type_in_the_mobile_number_after_selecting_the_country_code" time="24.592"/>
  <testcase classname="Test_Supervisor" name="Verify_that_the_organization_name_can_be_selected_from_a_dropdown_menu" time="24.499"/>
  <testcase classname="Test_Supervisor" name="Verify_that_the_user_can_type_and_select_an_organization_name_if_it_is_not_listed_in_the_dropdown" time="20.291"/>
  <testcase classname="Test_Supervisor" name="Verify_that_an_error_message_is_displayed_if_the_save_button_is_clicked_when_mandatory_fields_are_not_filled" time="22.883"/>
  <testcase classname="Test_Supervisor" name="Verify_the_maximum_and_minimum_length_of_the_full_name" time="20.554"/>
  <testcase classname="Test_Supervisor" name="Verify_the_maximum_and_minimum_length_of_the_mobile_number_for_each_country_code" time="44.287"/>
  <testcase classname="Test_Supervisor" name="Enter_an_email_without_a_domain_and_verify_that_an_error_message_is_displayed" time="34.581"/>
  <testcase classname="Test_Supervisor" name="Verify_to_check_and_click_on_next_page_button_of_supervisor" time="19.017"/>
  <testcase classname="Test_Supervisor" name="check_to_update_supervisor_without_select_orgnization_name" time="24.167"/>
  <testcase classname="Test_Supervisor" name="check_and_update_superviaor_without_using_attherate" time="26.056"/>
  <testcase classname="Test_Supervisor" name="check_to_update_supervisor_without_enter_mandatory_field" time="22.192"/>
  <testcase classname="Test_Supervisor" name="Verify_to_update_supervisor_with_max_char_in_full_name_field" time="22.204"/>
  <testcase classname="Test_Supervisor" name="Verify_to_update_superviosr_with_invalid_number" time="22.323">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Invalid Mobile ] but found [Invalid Mobile No.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Invalid Mobile ] but found [Invalid Mobile No.]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at Test_Supervisor.Verify_to_update_superviosr_with_invalid_number(Test_Supervisor.java:711)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Verify_to_update_superviosr_with_invalid_number -->
  <testcase classname="Test_Supervisor" name="Verify_to_check_createsupervisor_that_are_already_exist" time="28.484"/>
  <testcase classname="Test_Supervisor" name="Create_Supervisor_with_valid_email" time="24.922"/>
</testsuite> <!-- Default test -->
